app/src/features/pricing/components/PremiumFeature/components/RequestFeatureModal/index.tsx

```diff
@@ -192,6 +192,7 @@ export const RequestFeatureModal: React.FC<RequestFeatureModalProps> = ({
                     <RQButton
                       type="primary"
                       onClick={() => {
+                        trackUpgradeOptionClicked("checkout_billing_teams");
```

Comment:
create constants for this.

Also create constants for all the options that are being passed here. For eg. I see `use_for_free_now` being used more than once

Reply:
++